### ISSUE TITLE
Turning off RLV should turn off the Relay

### ISSUE THREAD

**Issue by Slanador** [OpenCollarTeam/OpenCollar#1071]
Using version 8.2.x, I'm expecting that if I turn off RLV, that the Relay (which is at least from a user's perspective a part of the RLV feature) should also be turned off. Instead, the Relay stays on.

Suggestion
a) Have the RLV Relay its own seperate item (maybe not ideal, it's a bit crowded)
b) When turning off RLV features, also turn off the Relay.

**Comment by Slanador** [comment]
Part of this is also user error: I wanted to turn off the Relay, but instead I turned off the RLV. I assumed that that would turn off the Relay also, because I wanted to use a multirelay instead, like DEM. In my defense, I think it's intuitive to assume that Relay is a subfeature of RLV.

**Comment by SilkieSabra** [comment]
I agree that it is intuitive to assume the relay is a sub feature of RLV.  The reasons it is not have to do with the role of the relay which is basically a benign parasitic life form in the collar.  It has its own system of permissions, so it can be triggered separately from collar access.  Turning off the relay when you turn off RLV would lose this feature.  We've tried having the relay as separate item linked to the collar similarly to an add-on.  That leaves moving the relay out of the RLV menu to its own link in the front page, In my opinion this might reduce some confusion but not sure it's worth it.  Anyone else have some input here?
